### PR TITLE
Add styling props

### PR DESCRIPTION
**Pull request by tafelito**
This PR adds styling props to the container, cropping area and image

### CODE REVIEW

**Comment by ValentinH** [comment]
In order to use those props, you need to pass a style object right? What I had in mind was more something like `containerClass`, `cropAreaClass`, etc... Or a `classes` property that would take an object like this:
```
{
  container: 'xyz',
  cropArea: 'abc'
}
```

This would be simpler to integrate with any styling solution, CSS-in-JSS or not.

**Comment by ValentinH** [comment]
In order to use those props, you need to pass a style object right? What I had in mind was more something like `containerClass`, `cropAreaClass`, etc... Or a `classes` property that would take an object like this:
```
{
  container: 'xyz',
  cropArea: 'abc'
}
```

This would be simpler to integrate with any styling solution, CSS-in-JSS or not.

**Comment by tafelito** [comment]
I don't use classes. In fact I'm using react-native-web so I wouldn't be able to pass a class or a className. But either way, that could be added as well so you could either pass a class or a style and it will work in both cases

What do you think?

**Comment by ValentinH** [comment]
I understand. On the same idea as the one I suggested with classes, could you group all of them under a `styles` or `customStyle` property that would take an key/value map of styles?

Also could you please update the ReadMe so other people know how to use this new feature 🙂

**Comment by tafelito** [comment]
I haven't really used emotion or styled-components so what would be the best way to merge the classes with the default styles in this case?

File: src/index.js [hunk 1]
```diff
@@ -268,6 +270,7 @@ class Cropper extends React.Component {
             transform: `translate(${x}px, ${y}px) scale(${zoom})`,
           }}
           imageStyle={imageStyle}
+          className={cropAreaClassName}
```
Comment (ValentinH) [inline src/index.js]: I think it should be `imageClassName` here 😄

File: src/index.js [hunk 2]
```diff
@@ -279,6 +282,7 @@ class Cropper extends React.Component {
             }}
             data-testid="cropper"
             cropAreaStyle={cropAreaStyle}
+            className={imageClassName}
```
Comment (ValentinH) [inline src/index.js]: Opposite here: should be `cropAreaClassName`

File: src/index.js [hunk 3]
```diff
@@ -265,6 +269,8 @@ class Cropper extends React.Component {
           style={{
             transform: `translate(${x}px, ${y}px) scale(${zoom})`,
           }}
+          imageStyle={imageStyle}
+          className={cropAreaClassName}
```
Comment (ValentinH) [inline src/index.js]:
```suggestion
          className={imageClassName}
```

File: src/index.js [hunk 4]
```diff
@@ -275,6 +281,8 @@ class Cropper extends React.Component {
               height: this.state.cropSize.height,
             }}
             data-testid="cropper"
+            cropAreaStyle={cropAreaStyle}
+            className={imageClassName}
```
Comment (ValentinH) [inline src/index.js]:
```suggestion
            className={cropAreaClassName}
```

**Comment by ValentinH** [comment]
Thanks for this PR and the adjustements while I was a bit slow to answer 👍

**Comment by tafelito** [comment]
close #16

**Comment by tafelito** [comment]
@ValentinH, when do you think you'll publish a new version?

**Comment by ValentinH** [comment]
Released as 1.6: https://github.com/ricardo-ch/react-easy-crop/releases/tag/v1.6.0

**Pull request merged**